### PR TITLE
[plugin.video.retrospect] v5.2.8+matrix.1

### PR DESCRIPTION
**Pull request by basrieter**
### Description

Matrix versoin of https://github.com/xbmc/repo-plugins/pull/3013

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      "icon": "studio040icon.png",
+      "category": "Regional",
+      "channelcode": "studio040",
+      "sortorder": 100,
+      "language": "nl",
+      "fanart": "studio040fanart.jpg",
+      "adaptiveAddonSelectable": true
+    }
+  ]
+}
```
Comment (enen92) [inline plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.json]: Missing line at the end of the file

File: plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+            self.baseUrl = "https://media.wos.nl/retrospect/"
+            self.recentUrl = "https://media.wos.nl/retrospect/studio040/studio040-latest.json"
+
+        else:
+            raise NotImplementedError("Channelcode '%s' not implemented" % (self.channelCode, ))
+
+        # setup the main parsing data
+        self._add_data_parser(self.mainListUri, json=True,
+                              name="Mainlist parser",
+                              preprocessor=self.add_other_items, match_type=ParserData.MatchExact,
```
Comment (enen92) [inline plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py:90]: If his is shown in the UI it needs to be localised

File: plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+                              parser=[], creator=self.create_episode_item)
+
+        self._add_data_parser("https://media.wos.nl/stream/wos/live.php", json=True,
+                              name="Live stream parser",
```
Comment (enen92) [inline plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py:94]: Same

File: plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py (hunk 2 above)
Comment (basrieter) [reply plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py:90]: It is it not. Just in the logfile.

File: plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py (hunk 3 above)
Comment (basrieter) [reply plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py:94]: Same here.

File: plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.json (hunk 1 above)
Comment (basrieter) [reply plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.json]: Let me fix that.

**Comment by basrieter** [comment]
Fixed it @enen92

**Review by enen92 — APPROVED** [review]

**Pull request merged**